src/components/Form/Step3/DirectDebit/DirectDebit.js

```diff
@@ -36,17 +57,51 @@ const DirectDebit = () => {
         </h3>
         <p>
           This is the <strong>6-digit number</strong> beginning with a{' '}
-          <strong>6</strong>. It is shown next to every payment to WMCA for your
-          Direct Debit on your bank statement.
+          <strong>6</strong>.
+        </p>
+        <p>
+          It is shown next to every payment to WMCA for your Direct Debit on
+          your bank statement.
         </p>
       </legend>
       <Input
-        className="wmnds-col-1-2 wmnds-col-sm-1-5"
+        className={`wmnds-col-1-2 wmnds-col-sm-1-5 ${
+          noDDNumber && style.disabled
+        }`}
         name="DirectDebitNumber"
         label={label}
         inputmode="numeric"
-        customValidation={customValidation}
+        fieldValidation={ddNumValidation}
+        disabled={noDDNumber}
       />
+      <div className="wmnds-fe-group">
+        <details className="wmnds-details">
+          <summary className="wmnds-link">
+            I can&rsquo;t find my Direct Debit reference
+          </summary>
+          <div className="wmnds-details__content">
+            <p>
+              Check for an email when you originally set up your Direct Debit.
+            </p>
+            <p>
+              <strong>I still can&rsquo;t find the number</strong>
+            </p>
+            <p>
+              We&rsquo;ll try to match your details and we&rsquo;ll get in touch
+              if we need any more information.
+            </p>
+            <InputCheckbox
+              name="NoDirectDebitNumber"
+              type="checkbox"
+              fieldValidation={register({
+                required: false,
+              })}
+              handleChange={() => setNoDDNumber(!noDDNumber)}
+              labelElement={checkBoxLabel}
+            />
+          </div>
+        </details>
```

Comment:
This needs a polyfill for IE 11. I have already done this in the design system...
https://github.com/wmcadigital/wmn-design-system/blob/master/src/www/assets/vendor/js/polyfills/details-element.js

Just recreate that JS file in your app and include it in your index.js on line 4 and all should be good.